docs/docs/tutorial/creating-a-source-plugin/part-0/index.mdx

```diff
@@ -0,0 +1,116 @@
+
+Source plugins are an integral part to Gatsby's [GraphQL data layer](/docs/reference/graphql-data-layer/) as they are responsible for bringing data into that layer. They play a key role in hiding complex pieces of software (e.g. fetching and normalizing data from a remote API) behind an easy-to-use plugin. This also means that they have to be reliable, performant, and use best practices behind the scenes.
+
+There is already a wealth of [existing source plugins](/plugins?=gatsby-source) in the Gatsby ecosystem you can use, but there may come a time when you want to create your own source plugin that doesn't already exist. If that sounds like a situation you've found yourself in, this tutorial is for you! Otherwise if you're curious to learn more about how Gatsby's source plugins work, you'll also find useful information here.
+
+<Announcement>
+
+This tutorial is indended to be an **additive guide**. Each part builds up on each other so we recommend you going through it from start to finish. However, if you want to revisit a specific topic each part should have all the necessary information on its own to be informative.
+
+You'll have a functioning source plugin at the end of part 2. The later parts are improving the plugin, adding functionalities, and explaining advanced topics. So while you can already publish what you have after part 2, we highly recommend going through all parts to learn how you can further improve your plugin.
+
+</Announcement>
+
+By the end of this part of the tutorial, you will:
+
+- Know what background knowledge is assumed throughout the tutorial
+- Understand what a source plugin is and why you would want to create one
+- Install the necessary software
+
+## Background knowledge
+
+This tutorial assumes a basic understanding of Gatsby, [GraphQL](https://graphql.org/), [Node.js](https://nodejs.org/en/), and [TypeScript](https://www.typescriptlang.org/).
+
+<Announcement>
+
+Don't worry, you don't need to be an expert with these! A high-level understanding of the basics should be enough. You'll pick up a lot through the course of the tutorial.
+
+</Announcement>
+
+In terms of Gatsby-related knowledge, you should first complete the [Getting Started Tutorial](/docs/tutorial/getting-started/) since this guide builds on the knowledge established there.
+
+In terms of the other technologies, you should be comfortable writing GraphQL queries, importing/exporting JavaScript modules, and making use of TypeScript types.
+
+## What is a source plugin?
+
+A plugin that adds additional data sources (content and data backends, e.g. CMSs like Contentful, WordPress, Drupal) to Gatsby that can then be queried by your pages and components. You can access these created nodes via Gatsby's universal GraphQL data layer.
```

Comment:
If it's the first instance of CMS, should this be: 
```suggestion
A plugin that adds additional data sources (content and data backends, e.g. content management systems like Contentful, WordPress, Drupal) to Gatsby that can then be queried by your pages and components. You can access these created nodes via Gatsby's universal GraphQL data layer.
```
Normally I would suggest content management system (CMS) and then just keep using CMS, but it's a little awkward to do here inside a parentheses already, so I can see why you went straight to the abbreviation.